### PR TITLE
News list and show

### PR DESCRIPTION
**Pull request by angel-afonso**
- feat: added news listing
- feat: added news list and view
- feat: added test cases
- feat: Updated newsView component and buttons in the markdown editor now furulan
- feat: Comments now works, testing is needed
- added in news comments
- removed logs

# News lists and view

Reason:
This is a crucial part of the requirements

Before pull request:
There are no way to view news

After pull request:
A list and a view section was added

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  singleQuote: true,
+  trailingComma: 'all',
+  singleQuote: true,
```
Comment (HectorZR) [inline .prettierrc.js:4]: Has the same rule here

File: src/screens/news/functions.js [hunk 2]
```diff
@@ -12,16 +12,14 @@ export function useFetchNews() {
...
         setLoading(false);
         setPosts(
           response?.ids?.map((id) => {
             return response?.data?.[id];
-          }) ?? []
+          }) ?? [],
         );
       })
       .catch((e) => {
-        console.log(e);
```
Comment (HectorZR) [inline src/screens/news/functions.js:24]: Good work here removing the `console.log()`

**Review by HectorZR — CHANGES_REQUESTED** [review]

File: .prettierrc.js (hunk 1 above)
Comment (angel-afonso) [reply .prettierrc.js:3]: done

Comment (HectorZR) [reply .prettierrc.js:3]: Nothing has changed in the PR

**Review by HectorZR — CHANGES_REQUESTED** [review]

File: .prettierrc.js (hunk 1 above)
Comment (angel-afonso) [reply .prettierrc.js:3]: Apply systems engineering

**Review by HectorZR — APPROVED** [review]
Good work I like the new structure to group components in their own folder exporting from an index

**Pull request merged**